README.md

```diff
@@ -2264,3 +2264,8 @@ harshithap2000-dai_deprivation"
 "sulamanshahbukhari_dino"
 
 ![](sulamanshahbukhari_dino.png)
+
+"etendra2501_dino_dino"
+
+![](https://github.com/etendra2501/dinosaurs/blob/etendra2501_dino_dino/etendra2501_dino_dino.png)
```

Comment:
Could you upload your dino as a file to the repo instead adding it as a link?

Reply:
done